### ISSUE TITLE
[nemo-emblems] - Emblems too large on max zoom of image previews

### ISSUE THREAD

**Issue by andrewrembrandt** [linuxmint/nemo-extensions#337]
Hi,
When increasing the zoom level in nemo (e.g. to preview images) the emblem size also increases - would it not be better to keep the emblem size smaller?
![image](https://user-images.githubusercontent.com/6534105/66912861-bac49100-f013-11e9-860c-67ac67cdb3c7.png)

[This is on Arch linux with Nemo 4.2.3-1 and nemo-emblems 4.2.0-1]
Many thanks (great file manager and extensions by the way)

**Comment by smurphos** [comment]
I wouldn't want any change to emblem behaviour applied to zoomed directories. It would break my (mis)use of the emblems feature as directory previews. 

![screenshot-window-2019-10-17-065309](https://user-images.githubusercontent.com/29017677/66981187-e8104e00-f0aa-11e9-830d-bf25ac01618e.png)

**Comment by claudiux** [comment]
@smurphos Your (mis)use interests me a lot! How did you do that?

**Comment by claudiux** [comment]
@smurphos You're a genius, you know?

**Comment by smurphos** [comment]
I wouldn't go that far.....but thank-you - I was quite pleased with that one.

**Comment by andrewrembrandt** [comment]
Interestingly, Dolphin handles the emblems (and image previews for a folder) separately. I'm not sure if that means anything for emblems in nemo however: 
![image](https://user-images.githubusercontent.com/6534105/67143291-4fc0c780-f26a-11e9-932f-51c80ec584b0.png)

**Comment by claudiux** [comment]
It would be cool if nemo did the same thing.

**Comment by smurphos** [comment]
Yep, it was Nemo's lack of dedicated folder contents preview functionality that prompted me to leverage the emblem functionality to emulate it.

**Comment by freder** [comment]
coming back to the [original issue](https://github.com/linuxmint/nemo-extensions/issues/337#issue-507775335) – is there a way to keep the emblem at a fixed size?
